src/app/InteractionModelEngine.h

```diff
@@ -164,6 +164,7 @@ class InteractionModelEngine : public Messaging::ExchangeDelegate
 
 void DispatchSingleClusterCommand(chip::ClusterId aClusterId, chip::CommandId aCommandId, chip::EndpointId aEndPointId,
                                   chip::TLV::TLVReader & aReader, Command * apCommandObj);
+CHIP_ERROR CheckIfClusterCommandExists(chip::ClusterId aClusterId, chip::CommandId aCommandId, chip::EndpointId aEndPointId);
```

Comment:
This needs to be documented.  Especially because it's intended to be implemented by ... something else, and the something else needs to understand the semantics it's implementing.